sysfs/vulnerability_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 The Prometheus Authors
```

Comment:
Nit, new files get the current year.

```suggestion
// Copyright 2023 The Prometheus Authors
```